volatility3/framework/plugins/linux/lsof.py

```diff
@@ -34,26 +36,45 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                                          optional = True)
         ]
 
-    def _generator(self, tasks):
-        symbol_table = None
-        for task in tasks:
-            if symbol_table is None:
+    @classmethod
+    def list_fds(cls,
+                 context: interfaces.context.ContextInterface,
+                 symbol_table: str,
+                 filter_func: Callable[[int], bool] = lambda _: False):
+
+        linuxutils_symbol_table = None  # type: ignore
+        for task in pslist.PsList.list_tasks(context, symbol_table, filter_func):
+            if linuxutils_symbol_table is None:
                 if constants.BANG not in task.vol.type_name:
                     raise ValueError("Task is not part of a symbol table")
-                symbol_table = task.vol.type_name.split(constants.BANG)[0]
+                linuxutils_symbol_table = task.vol.type_name.split(constants.BANG)[0]
```

Comment:
Will there be another symbol table used?  Not sure why this is now `linuxutils_...`?  No problem with it, just wondering why.  5;)